src/media/javax/microedition/media/PlayerImpl.java

```diff
@@ -854,6 +854,18 @@ private static long crcUtil(byte[] b) {
 		return crc.getValue();
 	}
 
+	public String getReadableImplementationType() {
+		if(sequence == null)
+			return "Empty player";
+		if(sequence instanceof Sequence)
+			return "JVM MIDI";
+		if(sequence instanceof Clip)
+			return "JVM clip";
+		if(sequence instanceof javazoom.jl.player.Player)
+			return "javazoom";
```

Comment:
MP3*

Reply:
It's stated in mime type column (`audio/mpeg` or `audio/mp3`). This column is intended to say which piece of code is responsible for you sound playing wrong (if it's playing wrong), and this time it's "javazoom" library.